Scripts/SetupDevService.bat

```diff
@@ -1,4 +1,6 @@
 @ECHO OFF
+CALL %~dp0\EnsureVFSDevShell.bat || EXIT /b 10
+
 IF "%1"=="" (SET "Configuration=Debug") ELSE (SET "Configuration=%1")
 
-sc create %Configuration%.GVFS.Service binPath=%~dp0\..\..\BuildOutput\GVFS.Service\bin\x64\%Configuration%\GVFS.Service.exe
\ No newline at end of file
+sc create %Configuration%.GVFS.Service binPath=%VFS_OUTPUTDIR%\GVFS.Service\bin\x64\%Configuration%\GVFS.Service.exe
```

Comment:
I didn't realize we had this script.  @sanoursa do you know if anyone\anything uses this script?  Or should we just remove it?

Reply:
I have never used them, and I can't find any other scripts that call them. I think they must have been added to help with testing changes to the built GVFS.Service without installing it, but I don't think they add any value anymore. I'd be good with deleting all of the `*DevService.bat` scripts.

Reply:
I will do that in the subsequent PR that also refactors the scripts dir.